engine-precompiles/src/native.rs

```diff
@@ -237,17 +244,24 @@ impl<I: IO> Precompile for ExitToNear<I> {
         context: &Context,
         is_static: bool,
     ) -> EvmPrecompileResult {
+        // ETH transfer input format: (21+ bytes)
+        //  - flag (1 byte)
+        //  - refund_address (20 bytes)
+        //  - recipient_account_id (N bytes)
+        // ERC20 transfer input format: (53+ bytes)
+        //  - flag (1 byte)
+        //  - refund_address (20 bytes)
+        //  - amount (32 bytes)
+        //  - recipient_account_id (N bytes)
         #[cfg(feature = "error_refund")]
         fn parse_input(input: &[u8]) -> Result<(Address, &[u8]), ExitError> {
-            if input.len() < 21 {
-                return Err(ExitError::Other(Cow::from("ERR_INVALID_INPUT")));
-            }
-            let refund_address = Address::try_from_slice(&input[1..21]).unwrap();
-            Ok((refund_address, &input[21..]))
+            is_valid_input_size(input, 21, None)?;
+            let refund_address = Address::try_from_slice(&input.get(1..21).unwrap()).unwrap();
```

Comment:
I'm just curious what's the motivation to replace indexing access with the `get` method? Anyway, you don't handle an Option the `get` method returns.

Reply:
I think the `get` method is redundant here, and you can use indexing access safely because you make a validation check by the `validation_input_size` function.

Reply:
We shouldn't be using unwrap, at all, anywhere ever. Production code should never have it period.